src/models/Customers.ts

```diff
@@ -47,6 +49,14 @@ const customersSchema: Schema = new Schema({
     type: Boolean,
     default: false,
   },
+  isBlacklisted: {
+    type: Boolean,
+    default: false,
+  },
+  score: {
+    type: Number,
+    default: 0,
+  },
   amountSpent: {
     type: Number,
     default: 0,
```

Comment:
the customer will also need to have warnings property right? Just like the employees